### PR TITLE
various fixes for Windows

### PR DESCRIPTION
**Pull request by dirk-thomas**
Also addresses #3.

These changes get us to the following state: http://54.183.26.131:8080/view/ros2/job/ros2_batch_ci_windows/184/

@esteve @tfoote @wjwwood Please review.

### CODE REVIEW

**Comment by tfoote** [comment]
lgtm

**Comment by esteve** [comment]
+1

File: launch/launch/launcher.py [hunk 1]
```diff
@@ -174,19 +173,21 @@ def _run(self):
...
-            # sending SIGINT to remaining processes
-            for index in all_futures.values():
-                p = self.task_descriptors[index]
-                if 'transport' in dir(p):
-                    self._process_message(p, 'signal SIGINT')
-                    try:
-                        p.transport.send_signal(signal.SIGINT)
-                    except ProcessLookupError:
-                        pass
+            # sending SIGINT to subprocess transport is not supported on Windows
```
Comment (wjwwood) [inline launch/launch/launcher.py]: Should we link to the appropriate issue in comments?

https://github.com/python/asyncio/issues/191
https://groups.google.com/forum/#!topic/python-tulip/pr9fgX8Vh-A

**Comment by wjwwood** [comment]
Other than comment +1

**Comment by dirk-thomas** [comment]
The subprocess returnvalue seems to not work on non-Windows. Therefore I readded the waitpid check for those platforms: 47bb7510e0634acde3ac9048b41a525ae83cdc86

**Pull request merged**